Octokit/Clients/RepositoriesClient.cs

```diff
@@ -86,6 +86,11 @@ public Task<Repository> Create(string organizationLogin, NewRepository newReposi
                     errorMessage,
                     StringComparison.OrdinalIgnoreCase))
                 {
+                    if (string.IsNullOrEmpty(organizationLogin))
+                    {
+                        throw new RepositoryExistsException(newRepository.Name, e);
+                    }
```

Comment:
In the previous version of the code `ExistingRepositoryWebUrl` was being set only if the owner is an organization. This is due to fact that we don't have access to the owner's name if it's an account and not an organization.